selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/NewWorkspace.java

```diff
@@ -116,7 +126,59 @@ public NewWorkspace(
     String QUICK_START_BUTTON_ID = "quick-start-button";
     String SINGLE_MACHINE_BUTTON_ID = "single-machine-button";
     String MULTI_MACHINE_BUTTON_ID = "multi-machine-button";
-    String ADD_OR_IMPORT_PROJECT_BUTTON_ID = "ADD_PROJECT";
+  }
+
+  public enum Stack {
+    BLANK("blank-default"),
+    JAVA("java-default"),
+    JAVA_MYSQL("java-mysql"),
+    DOT_NET("dotnet-default"),
+    ANDROID("android-default"),
+    CPP("cpp-default"),
+    CENTOS_BLANK("centos"),
+    CENTOS_GO("centos-go"),
+    CENTOS_NODEJS("nodejs4"),
+    CENTOS_WILDFLY_SWARM("wildfly-swarm"),
+    CENTOS_WITH_JAVA_JAVASCRIPT("ceylon-java-javascript-dart-centos"),
+    DEBIAN("debian"),
+    DEBIAN_LSP("debianlsp"),
+    ECLIPSE_CHE("che-in-che"),
+    ECLIPSE_VERTX("vert.x"),
+    GO("go-default"),
+    HADOOP("hadoop-default"),
+    JAVA_CENTOS("java-centos"),
+    JAVA_DEBIAN("java-debian"),
+    JAVA_THEIA_DOCKER("java-theia-docker"),
+    JAVA_THEIA_OPENSHIFT("java-theia-openshift"),
+    JAVA_MYSQL_CENTOS("java-centos-mysql"),
+    KOTLIN("kotlin-default"),
+    NODE("node-default"),
+    OPENSHIFT_DEFAULT("openshift-default"),
+    OPENSHIFT_SQL("openshift-sql"),
+    PHP("php-default"),
+    PHP_GAE("php-gae"),
+    PHP_5_6("php5.6-default"),
+    PLATFORMIO("platformio"),
+    PYTHON("python-default"),
+    PYTHON_2_7("python-2.7"),
+    PYTHON_GAE("python-gae"),
+    RAILS("rails-default"),
+    SELENIUM("selenium"),
+    SPRING_BOOT("spring-boot"),
+    TOMEE("tomee-default"),
+    UBUNTU("ubuntu"),
+    ZEND("zend");
+
+    private final String stackId;
+
+    Stack(String stackId) {
+      this.stackId = stackId;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
We doesn't need to overide `toString()` method here. _Getter_ method looks more natural here.
Also `stackId` can be rewritten to `id` without loosing initial meaning.